# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/serviceworkers/ServiceWorkerContainer.h
# Commit: 244df402d6eb
# Full Hash: 244df402d6ebc15d6845c4144245511021408835
# Author: Andrew Sutherland <asutherland@asutherland.org>
# Date: 2024-10-24 09:44:34
# Regressor Bug: 1131324
# File Overlap Count: 1
# Description:
#   Bug 1131324 - Expose ServiceWorkerContainer on WorkerNavigator. r=dom-worker-reviewers,webidl,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D213725
# ==============================================================================

diff -r 57f70ba5f27e -r 244df402d6eb dom/serviceworkers/ServiceWorkerContainer.h
--- a/dom/serviceworkers/ServiceWorkerContainer.h	Thu Oct 24 03:02:39 2024 +0000
+++ b/dom/serviceworkers/ServiceWorkerContainer.h	Thu Oct 24 03:02:40 2024 +0000
@@ -96,7 +96,8 @@
   // devtools console.
   nsIGlobalObject* GetGlobalIfValid(
       ErrorResult& aRv,
-      const std::function<void(Document*)>&& aStorageFailureCB = nullptr) const;
+      const std::function<void(nsIGlobalObject*)>&& aStorageFailureCB =
+          nullptr) const;
 
   struct ReceivedMessage;
 